libraries/AP_BoardConfig/AP_BoardConfig_CAN.cpp

```diff
@@ -172,8 +201,11 @@ void AP_BoardConfig_CAN::init()
             } else {
                 continue;
             }
-
-            _drivers[i]._driver->init(i);
+            if (_slcan_rt == 0) {
+                _drivers[i]._driver->init(i, true);
```

Comment:
Is this calling CanIface::init() ? If it is, then that should be an OperatingMode not a bool

Reply:
Its AP_CANProtocol derived to AP_UAVCAN, AP_KDECAN etc init.